internal/util/util.go

```diff
@@ -19,19 +19,87 @@ package util
 import (
 	"crypto/sha1"
 	"fmt"
+	"sort"
 
+	goyaml "gopkg.in/yaml.v2"
```

Comment:
Any reason for not using v3?

Reply:
I used the v2 which is being used in _"sigs.k8s.io/yaml"_ and in the v2, they support _MapSlice_ structure which is easy to handle to me.